tests/unit_tests/data_steward/validation/main_test.py

```diff
@@ -438,90 +438,88 @@ def test_process_hpo_ignore_dirs(
             self.assertTrue(filepath.startswith('SUBMISSION/'))
 
     @mock.patch('validation.main.StorageClient')
-    @mock.patch('gcs_utils.list_bucket')
+    @mock.patch('validation.main.list_bucket')
     @mock.patch('api_util.check_cron')
-    def test_copy_files_ignore_dir(self, mock_check_cron, mock_bucket_list,
+    def test_copy_files_ignore_dir(self, mock_check_cron, mock_list_bucket,
                                    mock_storage_client):
         """
         Test copying files to the drc internal bucket.
-
         This should copy anything in the site's bucket except for files named
         participant.  Copy_files uses a case insensitive match, so any
         capitalization scheme should be detected and left out of the copy.
         Anything else should be copied.  Mocks are used to determine if the
         test ran as expected and all statements would execute in a producstion
         environment.
-
         :param mock_check_cron: mocks the cron decorator.
         :param mock_bucket_list: mocks the list of items in the hpo bucket.
         :param mock_storage_client: mocks the StorageClient which has bucket and blob functionality.
         """
         # pre-conditions
-        mock_hpo_bucket = 'noob'
-        mock_drc_bucket = 'unit_test_drc_internal'
-        mock_bucket_list.return_value = [{
-            'name': 'participant/no-site/foo.pdf',
+        mock_hpo_bucket = mock.MagicMock()
+        mock_drc_bucket = mock.MagicMock()
+
+        mock_list_bucket.return_value = [{
+            'name': 'participant/site_1/person.csv',
         }, {
-            'name': 'PARTICIPANT/siteone/foo.pdf',
+            'name': 'PARTICIPANT/site_2/measurement.csv',
         }, {
-            'name': 'Participant/sitetwo/foo.pdf',
+            'name': 'Participant/site_3/person.csv',
         }, {
             'name': 'submission/person.csv',
         }, {
             'name': 'SUBMISSION/measurement.csv',
         }]
 
-        mock_client = mock.MagicMock()
-        mock_source_bucket = mock.MagicMock()
-        mock_destination_bucket = mock.MagicMock()
         mock_source_blob = mock.MagicMock()
+        mock_hpo_bucket = mock.MagicMock()
+        mock_drc_bucket = mock.MagicMock()
```

Comment:
Repeated pre-conditions already defined in lines 458 and 459.

Reply:
Sometimes you get something working and forget to clean up your mess. Committed.